rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -554,6 +566,10 @@ void game_list_frame::Refresh(const bool from_drive, const bool scroll_after)
 
 				add_disc_dir(game_dir);
 			}
+			else if (has_sfo)
+			{
+				m_path_list.emplace_back(game_dir);
+			}
 			else
 			{
 				game_list_log.trace("Invalid disc path registered for %s: %s", pair.first.Scalar(), pair.second.Scalar());
```

Comment:
this log message is kinda wrong now, since you don't really only care for discs

Reply:
Fixed.